micro-osd.sh

```diff
@@ -72,14 +74,13 @@ export CEPH_CONF=${DIR}/ceph.conf
 # start an osd
 ceph-mon --id ${MON_NAME} --mkfs --keyring /dev/null
 touch ${MON_DATA}/keyring
-ceph-mon --id ${MON_NAME}
+ceph-run ceph-mon -f --id ${MON_NAME} &
```

Comment:
I have found that this use of `ceph-run` does not appear to "work" in that it doesn't seem to consistently restart ceph-mon when it crashes. Since it doesn't seem to be serving much of a purpose, I suggest dropping this and if we need to work around ceph-mon crashing in the future we just deal with it later in a different PR.

Reply:
done